cpp/ql/src/semmle/code/cpp/dataflow/internal/DataFlowImpl.qll

```diff
@@ -1028,6 +1058,17 @@ private predicate consCandFwd(Content f, AccessPathFront apf, Configuration conf
   )
 }
 
+pragma[noinline]
+private predicate flowCandFwdRead(
+  Content f, Node node, AccessPathFront apf, boolean fromArg, Configuration config
```

Comment:
Could you switch the order of `apf` and `fromArg` to match the order in `flowCandFwd`?